tokio/src/runtime/dump.rs

```diff
@@ -30,14 +30,167 @@ pub struct Task {
     trace: Trace,
 }
 
+/// A backtrace symbol. This is similar to [backtrace::BacktraceSymbol],
+/// but is a separate struct to avoid public dependency issues.
+///
+/// This struct is guaranteed to be pure data and operations involving
+/// it will not call platform functions that take an unpredictable amount
+/// of time to finish.
+#[derive(Clone, Debug)]
+pub struct BacktraceSymbol {
+    name: Option<Vec<u8>>,
+    name_demangled: Option<String>,
```

Comment:
I think a boxed slice would be enough here.

```suggestion
    name: Option<Box<[u8]>>,
    name_demangled: Option<Box<str>>,
```